File: Python/3170.py
b = int(input())
g = int(input())

if g % 2 == 1:
    g -= 1

falta = int((g/2) - b)

if falta > 0:
    print(f"Faltam {falta} bolinha(s)")
else:
    print("Amelia tem todas bolinhas!")

File: C/3170.c
#include <stdio.h>
 
int main() {
 
    int b, g, falta;
    
    scanf("%i", &b);
    scanf("%i", &g);
    
    falta = (g/2) - b;
    
    if(falta > 0){
        printf("Faltam %i bolinha(s)\n", falta);
    }else{
        printf("Amelia tem todas bolinhas!\n");
    }
 
    return 0;
}